internal/appsec/rule.go

```diff
@@ -8,6 +8,6 @@
+// Static recommended AppSec rule 1.2.6
+// Source: https://github.com/DataDog/appsec-event-rules/blob/1.2.6/v2/build/recommended.json
```

Comment:
`v2` needs to be removed from here as well: `_tools/rules-updater/escaper/template.txt`